src/components/accordion/_template.njk

```diff
@@ -1,6 +1,6 @@
 {% from "components/details/_macro.njk" import onsDetails %}
 
-<div class="accordion">
+<div id="{{params.id}}" class="accordion {{ params.classes }}">
```

Comment:
Could add an if to avoid whitespace when no classes are passed
```suggestion
<div id="{{params.id}}" class="accordion{{ ' ' + params.classes if params.classes }}">
```